src/runtime/pay-client.js

```diff
@@ -174,6 +174,22 @@ class PayClientBindingSwg {
 
   /** @override */
   start(paymentRequest, options) {
+    if (options.forceRedirect) {
```

Comment:
PayClientBindingSwg is the obsolete lib and only used by AMP.

Reply:
Ok, I've updated both objects now.  Thank you!